packages/smart-contracts/scripts-create2/compute-from-list.ts

```diff
@@ -0,0 +1,38 @@
+import { computeCreate2DeploymentAddress } from './compute-one-address';
+import { create2ContractDeploymentList } from './utils';
+import { HardhatRuntimeEnvironmentExtended } from './types';
+
+export const computeCreate2DeploymentAddressesFromList = async (
```

Comment:
We can reduce a bit the number of files by merging this declaration with `computeCreate2DeployementAddress`, no need to have 1 file per method.